NEWS.md

```diff
@@ -11,6 +11,7 @@
 ### RStudio Workbench
 
 * Added support for setting the `subPath` on Kubernetes sessions using `KubernetesPersistentVolumeClaim` mounts in `/etc/rstudio/launcher-mounts` (Pro #2976).
+* Fixed custom shortcuts not appearing correctly in menus 
```

Comment:
I recommend opening a new issue to track the remaining behaviors not addressed by this fix (e.g. tooltips). Then narrow down the existing issue to only cover the menus, and reference it here in the news entry. Also QA only tests issues, not PRs, so having an issue that covers what this PR fixes while tracking the unfixed parts elsewhere is ideal.

Reply:
I have opened new issue https://github.com/rstudio/rstudio/issues/9915 to track this, and changed the PR description to mention it.